interview-questions.md

```diff
@@ -8,48 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is a markup extension to Javascript. It is similar to HTMl, but it is not as JSX is for React and was created by React. 
 
-Researched answer:
+Researched answer: JSX is a markdown syntax extension to Javascript and it is combining the logic and the markup together. One syntax difference between the two are the empty fragments and also, you must return every parent element. Tags can also self close such as a component call, unlike HTML everything needs an opening and closing tag. 
+
+<!-- Sources from https://www.freecodecamp.org/news/html-vs-jsx-whats-the-difference/#:~:text=HTML%20is%20a%20very%20important,a%20syntactic%20sugar%20for%20React. -->
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-Your answer:
+Your answer: Yarn stands for Yet Another Resource Negotiator and is the file manager for Javascript. I believe node modules are modified. 
 
-Researched answer:
+Researched answer: I'm not exactly sure what the question means, but when yarn is added there a bunch of files that are added such as the node modules which are all the dependencies that are needed for React App. 
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: An anonymous function is an empty function that we need when we don't have a return statement for it.
 
-Researched answer:
+Researched answer: Anyonymous functions are functions without a name. It seems these are used when we don't have a need for the function at the moment so instead we use a log to show the variable.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: State values are where the values for the data are stored and props are how we access that data. 
+
+Researched answer: I've found that state values cannot be accessed or modified outside of the componenet and that props make componenets reuseable by giving the ability to get data from the parent componenent in the form of props. 
 
-Researched answer:
+(I think I was close?)
```

Comment:
You were close!  Props allow us to access state values in other components. state can be accessed in the component which it is stored (usually App.js)